testsuite/forge/src/backend/mod.rs

```diff
@@ -2,7 +2,7 @@
 // SPDX-License-Identifier: Apache-2.0
 
 mod local;
-pub use local::*;
+pub use local::{node as LocalNode, *};
 
 mod k8s;
-pub use k8s::*;
+pub use k8s::{node as K8sNode, *};
```

Comment:
Let's fix this by fixing what is publicly exported from the local and k8s modules. As in, don't mark the module `node` public instead only export `LocalNode` and `K8sNode` individually.